racetime/static/racetime/script/race.js

```diff
@@ -86,10 +127,10 @@ Race.prototype.createMessageItem = function(message, server_date, mute_notificat
     var timestamp = ('00' + posted_date.getHours()).slice(-2) + ':' + ('00' + posted_date.getMinutes()).slice(-2);
 
     var $li = $(
-        '<li>'
-        + '<span class="timestamp">' + timestamp + '</span>'
-        + '<span class="message"></span>'
-        + '</li>'
+        '<li>' +
+        '<span class="timestamp">' + timestamp + '</span>' +
+        '<span class="message"></span>' +
+        '</li>'
```

Comment:
I disagree with your linter here. 😉 Operators should go at the start of the line, for reasons noted in [PEP 8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) (yes I know this isn't actually Python code)

Reply:
Yeah that's fair. I should have rolled with pep8